src.gen/@amzn/codewhisperer-streaming/src/models/models_0.ts

```diff
@@ -1019,6 +1019,8 @@ export interface RelevantTextDocument {
    * @public
    */
   documentSymbols?: (DocumentSymbol)[] | undefined;
+  startLine?: number;
```

Comment:
consider moving these out from src.gen because code in src.gen are supposed to be auto generated. Next time it updates, these 2 fields will be gone